src/en/send.md

```diff
@@ -94,61 +94,106 @@ You can leave out this argument if your service only has one reply-to email addr
 
 ::: warning Enabling this feature
 
-Sending files by email is not enabled by default. To turn on this feature, [contact us](https://notification.canada.ca/contact).
+Sending files by email is an API-only feature. To turn on this feature, [contact us](https://notification.canada.ca/contact).
 :::
 
-To send a file by email, you'll need to add a placeholder to the template then upload a file. The placeholder will contain a secure link to download the file. 
+### File types and size requirements
+You can upload PDF, CSV, .jpeg, .png, .odt, .txt, .rtf,  Microsoft Excel and Microsoft Word Document files. Your email including the file must be smaller than 10MB.
 
-The links are unique and unguessable. GC Notify cannot access or decrypt your file.
+[Contact us](https://notification.canada.ca/contact) if you need to send other file types.
 
-### Add contact details to the file download page
+### Sending methods
 
-To turn on this feature, [contact us](https://notification.canada.ca/contact).
+You can send files in two ways on GC Notify:
 
-### Add a placeholder to the template
+1. directly attached to emails
+1. through a unique link to a web interface
 
-1. [Sign in to GC Notify](https://notification.canada.ca/sign-in).
-1. Go to the __Templates__ page and select the relevant email template.
-1. Select __Edit__.
-1. Add a placeholder to the email template using double brackets. For example:
+You are able to control how files are delivered to recipients on every
+API call. GC Notify cannot access or decrypt your files.
 
-"Download your file at: ((link_to_file))"
+::: tip Choosing a sending method
 
-### Upload your file
+People generally expect files to be directly attached to emails. With that in mind, it’s not uncommon to see attachments being blocked by security rules or email providers. Use the unique link method to prevent your attachments from being blocked.
 
-::: tip File types and size requirements
-You can upload PDF, CSV, .jpeg, .png, .odt, .txt, .rtf,  Microsoft Excel and Microsoft Word Document files. Your file must be smaller than 10MB.
+Before choosing a sending method, perform tests to see what works best for your use case.
 
-[Contact us](https://notification.canada.ca/contact) if you need to send other file types.
 :::
 
-You’ll need to convert the file into a string that is base64 encoded.
+### Upload your file
+
+To send files, pass a dictionary of arguments in the `personalisation` argument. Pass this dictionary to the placeholder key if it’s present in your template or use a name of your choice.
```

Comment:
Is there a way to say this without using arguments twice in the sentence?

Reply:
What does `or use a name of your choice` mean here? Does that mean they can use a different name than the placeholder or that they can name the file?